views/js/runner/plugins/content/accessibility/keyNavigation.js

```diff
@@ -465,11 +552,15 @@ define([
                             self.groupNavigator.focus();
                         }
                     });
-
-                    self.enable();
                 })
                 .on('unloaditem', function () {
                     self.disable();
+                })
+                .on('contentLinearTabNavigation', function() {
+                    state.contentNavigatorType = 'linear';
+                })
+                .on('contentDefaultTabNavigation', function() {
```

Comment:
The idea of having event to set the mode is great. However, I have a concern regarding the naming. By convention we prefer names in lowercase, with verbs.
Since this is mode change, it could also be a setter like event: `on('setkeynavigationmode', mode)`, in order to prevent to have to much events to managed at the end.

Reply:
done